DEVELOPMENT.adoc

```diff
@@ -0,0 +1,37 @@
+= Development
+
+== Running the formatter
```

Comment:
I think it would be helpful to have this be a `Makefile` target which depends on the two actions described below.

Reply:
We can then apply the same approach to https://github.com/apple/pkl-swift-examples